selfdrive/car/toyota/carcontroller.py

```diff
@@ -38,7 +38,16 @@ def update(self, enabled, active, CS, frame, actuators, pcm_cancel_cmd, hud_aler
       # offset for creep and windbrake
       pedal_offset = interp(CS.out.vEgo, [0.0, 2.3, MIN_ACC_SPEED + PEDAL_TRANSITION], [-.4, 0.0, 0.2])
       pedal_command = PEDAL_SCALE * (actuators.accel + pedal_offset)
-      interceptor_gas_cmd = clip(pedal_command, 0., MAX_INTERCEPTOR_GAS)
+      # Prius, Kluger and most LSS Lexus have full speed ACC, but do not automatically resume
+      # Send pedal briefly if car is in standstill and openpilot wants to accelerate to resume ACC
+      # Pedal command is only sent very briefly, so no need to differentiate between Kluger and other cars
+      if CS.CP.carFingerprint in [CAR.PRIUS, CAR.RAV4H, CAR.HIGHLANDER, CAR.HIGHLANDERH, CAR.LEXUS_CTH, CAR.LEXUS_ESH, CAR.LEXUS_NX, CAR.LEXUS_NXH, CAR.LEXUS_RX, CAR.LEXUS_RXH]:
+        if (actuators.accel > 0.1) and CS.pcm_acc_status == 7:
```

Comment:
this should use cruiseState.standstill. we should probably remove the condition in carState and always set it